sass/directives/_headers.scss

```diff
@@ -60,39 +60,14 @@
   }
 }
 
-@mixin header-hero-product($background-image-large, $background-image-medium: nil) {
+@mixin header-hero {
   position: relative;
-  min-height: $header-height-product;
   overflow: hidden;
   background-color: $base-background-color;
-  background-image: url(asset_path(#{$background-image-large}));
-  background-size: auto $header-height-product;
-  background-position: right bottom;
-  background-repeat: repeat no-repeat;
-
-  @media only screen and (max-width: $medium-screen-max) {
-    background-image: url(asset_path(#{$background-image-medium}));
-  }
 
   @media only screen and (max-width: $small-screen-max) {
     @include display(flex);
     @include align-items(center);
-    min-height: unset;
```

Comment:
Well I kept the `min-height: unset` within `header-hero-product` mixin instead of pulling it into the generic `header-hero` mixin since I was unsure if the various headers already had a preset min-height at `$small-screen-max`. But i guess it wouldn't matter since the class would just override that property. 

I can move it into `hero-header` if you'd like